pdns/pdns_recursor.cc

```diff
@@ -186,7 +186,7 @@ static size_t s_maxUDPQueriesPerRound;
 static uint64_t g_latencyStatSize;
 static uint32_t g_disthashseed;
 static unsigned int g_maxTCPPerClient;
-static unsigned int g_networkTimeoutMsec;
+unsigned int g_networkTimeoutMsec;
```

Comment:
Just a nit, but would you mind moving the variable below, with the other non-static ones?